libraries/AP_Motors/AP_MotorsHeli.h

```diff
@@ -48,8 +48,8 @@ class AP_MotorsHeli : public AP_Motors {
 public:
 
     /// Constructor
-    AP_MotorsHeli( uint16_t         loop_rate,
-                   uint16_t         speed_hz = AP_MOTORS_HELI_SPEED_DEFAULT) :
+    AP_MotorsHeli(uint16_t         loop_rate,
```

Comment:
I differ from other ways of having many variable types and variable white spaces. I think that it is better to combine with others. How is that?